### PR TITLE
Describing what could be done to make a text index work.

### PR DESCRIPTION
**Pull request by arnefischereit**

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc [hunk 1]
```diff
@@ -70,6 +70,20 @@ In summary, `TEXT` indexes support the following predicates:
 |substring search| `CONTAINS`
 |===
 
+In some cases it is not possible for the system to determine an expression to be of type string.
```
Comment (renetapopova) [inline cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc]:
```suggestion
In some cases, the system cannot determine an expression of a type string.
```

**Review by Dunfluff — APPROVED** [review]
LGTM

File: cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc [hunk 2]
```diff
@@ -70,6 +70,20 @@ In summary, `TEXT` indexes support the following predicates:
...
+
+[source, cypher, role="noheader"]
+----
+MATCH (n:Label) WHERE n.prop = $param
+----
+
+In such cases the author of the query can modify the query to provide this information. Depending on how values which are not of type string should be treated, there are two options:
```
Comment (renetapopova) [inline cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc]:
```suggestion
Such queries can be modified to provide this information. 
Depending on how values that are not of type String should be treated, there are two options:
```

**Review by renetapopova — APPROVED** [review]
LGTM, thanks @arnefischereit! I have two minor comments.

File: cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc (hunk 1 above)
Comment (arnefischereit) [reply cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc]: That does not cover what this should express. What do you think about:

```suggestion
In some cases, the system cannot determine whether an expression is of type string.
```

File: cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc (hunk 2 above)
Comment (arnefischereit) [reply cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc]: I like that. I am just curious whether you would capitalise "string" in this context. Then we should probably also do it in the first case.

File: cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc (hunk 1 above)
Comment (renetapopova) [reply cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc]: Sounds good to me, yes.

File: cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc (hunk 2 above)
Comment (renetapopova) [reply cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc]: Aaah, that's a leftover. First, I capitalized all of them, but then I saw that in the example, it's represented as `"string"`, so I removed the capitalization, or I thought I had removed it. 😄

**Comment by arnefischereit** [comment]
@renetapopova:
I included your suggestions plus an extra sentence in 79207dc. If you approve, you can merge this PR.

**Comment by renetapopova** [comment]
Looks good to me. I'll wait for the artefacts and see how it renders. Thanks, @arnefischereit!

**Pull request merged**